src/main/java/com/example/onlinebookstore/controller/BookController.java

```diff
@@ -1,8 +1,8 @@
 package com.example.onlinebookstore.controller;
 
-import com.example.onlinebookstore.dto.BookDto;
-import com.example.onlinebookstore.dto.BookSearchParametersDto;
-import com.example.onlinebookstore.dto.CreateBookRequestDto;
+import com.example.onlinebookstore.dto.bookdtos.BookDto;
```

Comment:
`dto.bookdtos`? 
'dto' is duplicated. And do we use the plural in package names?
`dto.book` is better